tests/models/test_amp.py

```diff
@@ -74,7 +74,7 @@ def _assert_autocast_enabled(self):
     "accelerator",
     [
         None,
-        pytest.param("dp", marks=pytest.mark.skip("dp + amp not supported currently")),  # TODO
+        pytest.param("dp", marks=pytest.mark.skip("dp + amp not supported on CPU currently")),  # TODO
```

Comment:
This test still fails for me, but since it only seems to happen on the CPU I thought it would be a good idea to call that out.

Reply:
Yes, this need some investigation. Thanks !